### PR TITLE
[#19] Fix relative Catalog Resolution

### LINKED ISSUE #19: Allow rewriting system ids for local files

### PR DESCRIPTION
**Pull request by laurentschoelens**
Fixes #19 having a "RelativeCatalog" that adds automatically catalog entries for `REWRITE_SYSTEM` that starts with ".."

### CODE REVIEW

**Review by mattrpav — APPROVED** [review]
LGTM

File: maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJC2Mojo.java [hunk 1]
```diff
@@ -446,6 +446,19 @@ public boolean getDisableSystemIdResolution() {
...
+     * If 'true', the fix for issue #19 will be applied.
+     */
+    @Parameter(defaultValue = "false", property = "maven.xjc2.relativeCatalogResolution")
+    private boolean relativeCatalogResolution;
+
+    public boolean getRelativeCatalogResolution() {
+        return relativeCatalogResolution;
+    }
+
+    public void getRelativeCatalogResolution(boolean relativeCatalogResolution) {
```
Comment (michael-o) [inline maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJC2Mojo.java:459]: How? `get` for a setter?

File: maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJC2Mojo.java [hunk 2]
```diff
@@ -446,6 +446,19 @@ public boolean getDisableSystemIdResolution() {
...
         this.disableSystemIdResolution = disableSystemIdResolution;
     }
+    /**
+     * If 'true', the fix for issue #19 will be applied.
+     */
+    @Parameter(defaultValue = "false", property = "maven.xjc2.relativeCatalogResolution")
+    private boolean relativeCatalogResolution;
+
+    public boolean getRelativeCatalogResolution() {
+        return relativeCatalogResolution;
```
Comment (michael-o) [inline maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJC2Mojo.java:456]: Should be `is`, no?

Comment (laurentschoelens) [reply maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJC2Mojo.java:456]: There is already some boolean parameter with "get". It has no direct impact on resolution for maven / mojo.

**Pull request merged**